lib/cog/command/ambiguous_alias_checker.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Cog.Command.AmbiguousAliasChecker do
+  alias Cog.Repo
+  alias Cog.Queries.Alias
+  alias Cog.Models.Command
+  alias Piper.Command.Ast
+
+  def ambiguous_command?(%Ast.Invocation{command: "site:" <> _}),
+    do: false
+  def ambiguous_command?(%Ast.Invocation{command: "user:" <> _}),
```

Comment:
If we are treating `site` and `user` effectively as "bundles" for the purposes of alias and command resolution, then we should prevent "real bundles" named `site` and `user` from ever being created.

(Come to think of it, we should have already done that with `site` since we have a permission namespace with that name.)